runtime/src/main/java/org/corfudb/runtime/clients/LogUnitClient.java

```diff
@@ -2,46 +2,25 @@
 
 import com.codahale.metrics.MetricRegistry;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.codahale.metrics.MetricRegistry'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)